Oukitel/WP8PROEEA/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="config_defaultListenerAccessPackages">com.ssui.launcher3:com.android.calendar:com.ssui.push:com.ssui.gmodel:com.android.launcher3:com.google.android.setupwizard:com.google.android.apps.restore:com.google.android.projection.gearhead</string>
```

Comment:
This MUST be left to the ROM to decide, it's not related to hardware at all.

Reply:
Done.